modules/installation-and-upgrade/pages/container-deployment/suma/proxy-container-setup.adoc

```diff
@@ -107,11 +107,13 @@ The following procedure assumes you have added your Organization Credentials to
 . Fill in the fields for your Proxy host.
 . Select the Activation key created in the previous step from the dropdown.
 . Click btn:[+ Bootstrap].
-. Wait for the Bootstrap process to complete successfully. Check the menu:Systems[] list. The {salt} minion will be listed once complete.
-. Seleect the host from the menu:Systems[] list and apply all patches to update it.
+. Wait for the Bootstrap process to complete successfully. Check the menu:Salt[] menu and confirm the {salt} minion key is listed and accepted.
```

Comment:
```suggestion
. Wait for the Bootstrap process to complete successfully.
  Check the menu:Salt[] menu and confirm the {salt} minion key is listed and accepted.
```

Reply:
In the rendered version it is still in the same line, though.
![image](https://github.com/uyuni-project/uyuni-docs/assets/12104291/18d8e134-abb4-4f83-98e5-31fb4203eca2)